easybuild/easyconfigs/f/FastQ_Screen/FastQ_Screen-0.14.1-foss-2019b.eb

```diff
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'Tarball'
+
+name = 'FastQ_Screen'
+version = '0.14.1'
+
+homepage = 'http://www.bioinformatics.babraham.ac.uk/projects/%(namelower)s/'
+description = """FastQ Screen allows you to screen a library of sequences in FastQ
+ format against a set of sequence databases so you can see if the composition of the
+ library matches with what you expect."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/StevenWingett/FastQ-Screen/archive']
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-use-EB-Perl.patch']
+checksums = [
+    '42985e9edc3178008db3f5e914153609cdadb1b880ba2c9214fa1181ba71b4b3',  # v0.14.1.tar.gz
+    '509d91be7eeb3958c300f8973ad3fdd4dcfbd81a33c829f508e1e73049332094',  # FastQ_Screen-use-EB-Perl.patch
+]
+
+# Upstream hard-codes Perl path
```

Comment:
I'm guessing that `--inject-checksums` has moved this comment from where it originally was.